firefox_puppeteer/ui/windows.py

```diff
@@ -188,7 +188,7 @@ def __eq__(self, other):
 
     @property
     def closed(self):
-        """Returns closed state of the chrome window
+        """Returns closed state of the chrome window.
```

Comment:
Please see for example https://github.com/mozilla/firefox-ui-tests/pull/58/files#diff-296ca2194a2c0e54996caffeb9bbd8baL14 where you have removed the dots.